packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -425,7 +425,6 @@ Your packaged app may be larger than expected if you dont ignore everything othe
           error: tab.log.bind(tab),
           warn: tab.log.bind(tab),
         },
-        publicPath: '/',
```

Comment:
I believe this was originally required for hot reloading to work, maybe only set it when in dev mode?

Refs: https://github.com/electron-userland/electron-forge/commit/c8097c2f38fb7b6b28010f1abe8d8b2029f10011

Reply:
Indeed, hot reloading wasn't working during my session (I had to Ctrl-R to reload), this would explain why I didn't see the issue in #713 

This is with `@electron-forge/plugin-webpack@6.0.0-beta.41`, `webpack@4.35.3`.

Reply:
I won't know how to express it in TypeScript, but this seems to do the job in JavaScript and should implement your suggestion:
```
          output: Object.assign( this.isProd ? {} : {publicPath:'/'}, {
            path: _path.default.resolve(_this.baseDir, 'renderer'),
            filename: '[name]/index.js',
            globalObject: 'self',
          }),
```

Reply:
Typescript is quite similar but will work through spreading.

E.g.

```
{
   ...(a ? { prop: value } : {}),
  otherProp: otherValue,
}
```

Reply:
This is because object.assign is not typed but object spreading unions types

Reply:
Thank you so much for the help and pointers!